src/common/types/core.js

```diff
@@ -412,16 +412,26 @@ type Collaborators = {
     next_marker: ?string,
 };
 
-type Collaboration = {
-    accessible_by: {
-        id: number | string,
-        login: string,
-        name: string,
-        type: 'user',
-    },
+type AccessibleBy = {
```

Comment:
this type looks familiar. Does this exist already somewhere? `UserMini`? If not, maybe a name like `AccessibleByUser` will explain its provenance a bit more clearly.